resources/js/store/modules/session.js

```diff
@@ -13,8 +14,9 @@ const getters = {
   isAuthenticated: state => {
     return !$.isEmptyObject(state.currentUser);
   },
+
   settings: (state) => (setting) => {
-    return $.isEmptyObject(state.settings) || !(setting in state.settings) ? null : state.settings[setting];
+    return $.isEmptyObject(state.settings) || _.get(state.settings, setting);
```

Comment:
Wouldn't it be better to return undefined if state.settings is empty? Otherwise it could be that a boolean in a setting value would be accidentally assumed as false.